renderer.js

```diff
@@ -1436,6 +1475,8 @@ class SDFGRenderer {
             }
         }
 
+        this.overlay_manager.draw();
+
         this.on_post_draw();
```

Comment:
this is what on_post_draw was written for...

Reply:
Moving